tests/blackbox/data/namespace-scoped/kyma_function_syntax_error/scenario.yml

```
@@ -0,0 +1,47 @@
+id: kyma-function-syntax-error
+description: The Function has a syntax error in its JavaScript source code.
+user_query: Why is my Kyma Function not working?
+resource:
+  kind: Function
+  api_version: serverless.kyma-project.io/v1alpha2
+  name: my-function-0
+  namespace: kyma-function-syntax-error
+expectations:
+  - name: function_syntax_error
+    statement: points out that the Function has a syntax error in its source code and provides a description of the error.
+    categories:
+      - kyma
+      - problem_finding
+    complexity: 2
+  - name: function_correct_syntax
+    statement: provides an example of how to fix the Function JavaScript source code.
+    categories:
+      - kyma
+      - solution_finding
+    complexity: 2
+  - name: step_by_step_guide
+    statement: contains a step-by-step guide on how to fix the issue.
+    categories:
+      - solution_finding
+      - step-by-step_guidance
+    complexity: 1
+  - name: partial_fix
+    statement: provides a partial JavaScript code for the Function with the correct syntax.
+    categories:
+      - solution_finding
+    complexity: 2
+  - name: partial_yaml
+    statement: provides a partial yaml for the Function with the correct syntax for its JavaScript source code.
+    categories:
+      - kyma
+      - yaml
+      - solution_finding
+    complexity: 2
+  - name: complete_yaml
+    statement: provides a complete yaml for the Function with the correct syntax for its JavaScript source code.
+    categories:
+      - kyma
+      - yaml
+      - solution_finding
+      - completeness
+    complexity: 2
```

To be more in line with the other scenarios, let's set this up to 3
```suggestion
    complexity: 3
```

---

Done